resources/aws/rti_aws_uploader.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python3.7
```

Comment:
It would be nice if you could use the [`logging` module](https://docs.python.org/3/library/logging.html). You can add a `debug` mode to make easier fixing issues.

Reply:
Please @iblancasa take a look to the new changes.